src/SprykerEco/Zed/Payone/Business/Payment/RequestSender/PayoneBaseAuthorizeSender.php

```diff
@@ -68,7 +77,12 @@ public function performAuthorizationRequest(
         $this->standartParameterMapper->setStandardParameter($requestContainer, $this->standardParameter);
 
         $apiLogEntity = $this->initializeApiLog($paymentEntity, $requestContainer);
-        $rawResponse = $this->executionAdapter->sendRequest($requestContainer);
+        $rawResponse = $this->repository->getPreauthorizedPaymentByReference($requestContainer->getReference());
```

Comment:
Sender shouldn't check anything. pls move it `$this->repository->getPreauthorizedPaymentByReference($requestContainer->getReference());` to one level app before calling `performAuthorizationRequest`